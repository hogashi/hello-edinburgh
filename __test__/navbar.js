import React from 'react';
import renderer from 'react-test-renderer';

import Navbar from '../src/components/navbar';
import { testNameToKey } from 'jest-snapshot/build/utils';

test('Navbar', () => {
  const component = renderer.create(
    <Navbar message='message' />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapShot();
});
