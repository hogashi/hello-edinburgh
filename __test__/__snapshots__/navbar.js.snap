// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar 1`] = `
<nav>
  <div>
    <a
      className="home"
      href="/"
    >
      Edinburgh
    </a>
  </div>
  <div
    id="message"
  >
    message
  </div>
  <div>
    <a
      href="/auth/twitter"
    >
      login
    </a>
    <a
      href="/logout"
    >
      logout
    </a>
  </div>
</nav>
`;
